fabric-rendering-fluids-v1/src/testmod/java/net/fabricmc/fabric/test/client/rendering/fluid/FabricFluidRenderingTestModClient.java

```diff
@@ -39,9 +40,10 @@ public void onInitializeClient() {
 		FluidRenderHandlerRegistry.INSTANCE.setBlockTransparency(Blocks.SPRUCE_DOOR, true);
 		FluidRenderHandlerRegistry.INSTANCE.setBlockTransparency(Blocks.WARPED_DOOR, true);
 
-		// Red stained glass will have falling fluid textures to the side
+		// Red stained-glass will have falling fluid textures to the side
```

Comment:
(it's "stained glass" without a hyphen, especially in mc since this is how mojang spells it)